web/static/ts/utils/dom.ts

```diff
@@ -29,3 +29,28 @@ export function insertAfter(elem: Element, refElem: Element): Element {
 
     return refElem.parentElement.insertBefore(elem, refElem.nextElementSibling);
 }
+
+/**
+ * @param alert The alert caused by the User/System.
+ */
+export function alertBox(alert: alertMessage) {
+    const error = document.querySelector("#errorAlert");
+    const errorBtn = document.querySelector("#errorAlertButton");
+    if (alert === null) {
+        throw new TypeError("Can't display alert: alert element not found");
+    }
+    if (alert.severity == 0) {
```

Comment:
Comparse against the enum rather than its associated integer. It's clearer that way. When someone just sees "0" they don't know what it means.